galasa-parent/dev.galasa.framework.api.openapi/src/main/resources/openapi.yaml

```diff
@@ -2169,18 +2184,21 @@ components:
         trace:
           type: boolean
     UpdateRunStatusRequest:
+      type: object
       properties:
         status:
           type: string
           enum: ["queued","finished"]
         result:
           type: string
-          enum: ["","cancelled"]
+          enum: [nil,"cancelled"]
     User:
+      type: object
       properties:
         login_id:
           type: string
           description: The ID representing the user's username.
+          enum: [nil,"cancelled"]
```

Comment:
dont think this line should be here, as this object does not have hard coded values

Reply:
ah, shoot, I think I did a bad merge here, just wasn't sure what it was meant to be